app/controllers/posts_controller.rb

```diff
@@ -7,4 +7,32 @@ def show
     @user = User.find(params[:user_id])
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+
+    render :new, locals: { post: @post }
+  end
+
+  def create
```

Comment:
- [ ] Great work on your project, however when I click on the create post button, I get the following syntax error. kindly fix this.

Before button click
![image](https://user-images.githubusercontent.com/81932478/208249103-96fef4a1-fc81-4644-b672-a00ce3a678ed.png)

After
![image](https://user-images.githubusercontent.com/81932478/208249116-22f512a5-ee0d-4d12-ba5c-9b3b392f31c4.png)

### Note that this error occurs when I try to click on the `comments` and `likes` links

Reply:
That is strange. I do not have that error on my end 

https://user-images.githubusercontent.com/89058561/208250302-58944b33-f42a-41f7-941c-2fd19d709b92.mp4